code_snippets/api-metric-metadata-get.py

```diff
@@ -0,0 +1,12 @@
+from datadog import initialize, api
+
+options = {
+    'api_key': '9775a026f1ca7d1c6c5af9d94d9595a4',
+    'app_key': '87ce4a24b5553d2e482ea8a8500e71b8ad4554ff'
+}
+
+initialize(**options)
+
+metric = 'system.cpu.idle'
+
+print api.Metadata.get(metric_name=metric)
```

Comment:
Believe you want to remove the print here to avoid a syntax error, as this will evaluate to:

return print api.Metadata.get(metric_name=metric)